doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+                  "itemType": "BPM"
+                },
+                {
+                  "itemName": "KSession",
+                  "itemValue": "",
+                  "itemType": "BPM"
+                },
+                {
+                  "itemName": "MergeMode",
+                  "itemValue": "MERGE_COLLECTIONS",
+                  "itemType": "BPM"
+                },
+                {
+                  "itemName": "RuntimeStrategy",
+                  "itemValue": "SINGLETON",
+                  "itemType": "BPM"
+                }
+              ],
+              "messages": [],
+              "container-alias": "employeerostering"
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+----
+--
+CreateContainerCommand::
++
+--
+Creates a KIE container in the {KIE_SERVER}.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container to be created
+|Required
+
+|`releaseId`
+|GAV (group ID, artifact ID, version) data of the new KIE container
+|Required
+
+|`status`
+|Status to be set on the new KIE container (`STARTED`, `STOPPED`)
+|Optional
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "create-container" : {
+      "container" : {
+        "status" : null,
+        "messages" : [ ],
+        "container-id" : "command-script-container",
+        "release-id" : {
+          "version" : "1.0",
+          "group-id" : "com.redhat",
+          "artifact-id" : "Project1"
+        },
+        "config-items" : [ ]
+      }
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Container command-script-container successfully deployed with module com.redhat:Project1:1.0.",
+      "result": {
+        "kie-container": {
+          "container-id": "command-script-container",
+          "release-id": {
+            "version" : "1.0",
+            "group-id" : "com.redhat",
+            "artifact-id" : "Project1"
+          },
+          "resolved-release-id": {
+            "version" : "1.0",
+            "group-id" : "com.redhat",
+            "artifact-id" : "Project1"
+          },
+          "status": "STARTED",
+          "scanner": {
+            "status": "DISPOSED",
+            "poll-interval": null
+          },
+          "config-items": [],
+          "messages": [
+            {
+              "severity": "INFO",
+              "timestamp": {
+                "java.util.Date": 1538762455510
+              },
+              "content": [
+                "Container command-script-container successfully created with module com.redhat:Project1:1.0."
+              ]
+            }
+          ],
+          "container-alias": null
+        }
+      }
+    }
+  ]
+}
+----
+--
+GetContainerInfoCommand::
```

Comment:
single attribute containerId of type string that is required

Reply:
Done.